### ISSUE TITLE
Publish a new release?

### ISSUE THREAD

**Issue by qubidt** [gotson/komga#973]
### Describe your suggested feature

Is the codebase in a reasonable place to push a (pre-)release? I'm waiting on the fix for #961 to import my library. My instance is running in docker so I'm kind of stuck on the tagged releases. It would be greatly appreciated

### Other details

The current latest release is [v0.157.1](https://github.com/gotson/komga/releases/tag/v0.157.1).

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by qubidt** [comment]
merci!
